definitions/liferay-portlet-app_7_1_0.dtd

```diff
@@ -0,0 +1,861 @@
+portal.properties, and the property "monitoring.portlet.render.request" in
+portal.properties is set to true, then the portlet's render phase processing
+will be timed. If the execution time is longer than render-timeout, it will be
+recorded as a timeout request processing. The time unit is millisecond.
+-->
+<!ELEMENT render-timeout (#PCDATA)>
+
+<!--
+The default value of render-weight is 1. If set to a value less than 1, the
+portlet is rendered in parallel. If set to a value of 1 or greater, then the
+portlet is rendered serially. Portlets with a greater render weight have greater
+priority and will be rendered before portlets with a lower render weight.
+
+If the ajaxable value is set to false, then render-weight is always set to 1
+if it is set to a value less than 1. This means ajaxable can override
+render-weight if ajaxable is set to false.
+-->
+<!ELEMENT render-weight (#PCDATA)>
+
+<!--
+The default value of ajaxable is true. If set to false, then this portlet can
+never be displayed via Ajax.
+-->
+<!ELEMENT ajaxable (#PCDATA)>
+
+<!--
+Set the path of CSS that will be referenced in the page's header relative to the
+portal's context path.
+-->
+<!ELEMENT header-portal-css (#PCDATA)>
+
+<!--
+Set the path of CSS that will be referenced in the page's header relative to the
+portlet's context path.
+-->
+<!ELEMENT header-portlet-css (#PCDATA)>
+
+<!--
+Set the path of JavaScript that will be referenced in the page's header relative
+to the portal's context path.
+-->
+<!ELEMENT header-portal-javascript (#PCDATA)>
+
+<!--
+Set the path of JavaScript that will be referenced in the page's header relative
+to the portlet's context path.
+-->
+<!ELEMENT header-portlet-javascript (#PCDATA)>
+
+<!--
+Set the path of CSS that will be referenced in the page's footer relative to the
+portal's context path.
+-->
+<!ELEMENT footer-portal-css (#PCDATA)>
+
+<!--
+Set the path of CSS that will be referenced in the page's footer relative to the
+portlet's context path.
+-->
+<!ELEMENT footer-portlet-css (#PCDATA)>
+
+<!--
+Set the path of JavaScript that will be referenced in the page's footer relative
+to the portal's context path.
+-->
+<!ELEMENT footer-portal-javascript (#PCDATA)>
+
+<!--
+Set the path of JavaScript that will be referenced in the page's footer relative
+to the portlet's context path.
+-->
+<!ELEMENT footer-portlet-javascript (#PCDATA)>
+
+<!--
+Set name of the CSS class that will be injected in the DIV that wraps this
+portlet.
+-->
+<!ELEMENT css-class-wrapper (#PCDATA)>
+
+<!--
+Set the facebook-integration value to either "fbml" or "iframe". The default
+value is "iframe" because IFrame integration will work without requiring any
+changes to your code. See the Message Boards portlet for minor changes that were
+made to make it FBML compliant. Note that the Liferay tag libraries already
+output FBML automatically if a request is made by Facebook.
+
+Deprecated As of 7.1.0, with no direct replacement.
```

Comment:
Hey @JorgeFerrer I marked this element as deprecated because otherwise I think that we will be breaking backwards compatibility. However, I couldn't find any other deprecated element in this class, so I don't know if this is the appropriate way of deprecating the element.